src/features/culturalSurvey/pages/CulturalSurveyQuestions.tsx

```diff
@@ -7,8 +7,8 @@ import styled from 'styled-components/native'
 import { extractApiErrorMessage } from 'api/apiHelpers'
 import { CulturalSurveyAnswer, CulturalSurveyAnswerEnum, CulturalSurveyQuestionEnum } from 'api/gen'
 import { useAuthContext } from 'features/auth/context/AuthContext'
-import { useCulturalSurveyAnswersMutation } from 'features/culturalSurvey/api/useCulturalSurveyAnswers'
-import { useCulturalSurveyQuestions } from 'features/culturalSurvey/api/useCulturalSurveyQuestions'
+import { useCulturalSurveyAnswersMutation } from 'features/culturalSurvey/api/useCulturalSurveyAnswersMutation'
+import { useCulturalSurveyQuestionsQuery } from 'features/culturalSurvey/api/useCulturalSurveyQuestionsQuery'
```

Comment:
il me semblait qu'on s'était dit qu'on bougeait les queries dans le dossier queries

```suggestion
import { useCulturalSurveyAnswersMutation } from 'features/culturalSurvey/queries/useCulturalSurveyAnswersMutation'
import { useCulturalSurveyQuestionsQuery } from 'features/culturalSurvey/queries/useCulturalSurveyQuestionsQuery'
```

est-ce qu'il y a des raisons pour avoir une exception sur ce cas ?